### PR TITLE
Bug 1822852: Provided APIs are empty on Installed Operators list when all provided APIs are internal 

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @TheRealJon
/cc @benjaminapetersen @spadgett

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx [hunk 1]
```diff
@@ -279,6 +279,9 @@ export const NamespacedClusterServiceVersionTableRow = withFallback<
   const route = resourceObjPath(obj, referenceFor(obj));
   const uid = getUID(obj);
   const internalObjects = getInternalObjects(obj);
+  const getProvidedAPIs = providedAPIsFor(obj).filter(
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx:282]: This makes the var sound like it's a function. I'd call this `providedAPIs`

```suggestion
  const providedAPIs = providedAPIsFor(obj).filter(
```

**Comment by spadgett** [comment]
@cyril-ui-developer FYI, whitespace before the `:` in the PR title keeps the bot from finding the BZ

File: frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx (hunk 1 above)
Comment (cyril-ui-developer) [reply frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx]: > This makes the var sound like it's a function. I'd call this `providedAPIs`

@spadgett Done, thanks.

**Pull request merged**